crowbar_framework/app/models/provisioner_service.rb

```diff
@@ -177,5 +177,65 @@ def transition(inst, name, state)
     @logger.debug("Provisioner transition: exiting for #{name} for #{state}")
     [200, { name: name }]
   end
-end
 
+  def load_repository_bag
+    bag = Chef::DataBag.load("repositories") rescue nil
+    if bag.nil?
+      create_repository_bag
+    end
+    bag
+  end
+
+  def synchronize_repositories(platforms)
+    # create DataBag if it doesn't exist yet
+    load_repository_bag
+    platforms.each do |platform, repos|
+      repos.each do |reponame, active|
+        repo_object = Crowbar::Repository.where(platform: platform, repo: reponame).first
+        case active.to_i
+        when 0
+          db = Chef::DataBagItem.load("repositories", repo_object.id) rescue nil
```

Comment:
Style/RescueModifier: Avoid using `rescue` in its modifier form. (https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers)